python/ctsm/site_and_regional/tower_site.py

```diff
@@ -0,0 +1,413 @@
+"""
+This module includes the definition for the TowerSite class,
+which has NeonSite and Plumber2Site child classes. This class defines common
+functionalities that are in both NeonSite and Plumber2Site classes.
+"""
```

Comment:
Maybe mention the ability to add other child classes that could be done here? GenericTowerSite, or another set of random sites...

Reply:
Let's leave this off for now.